src/components/PostPageDetail/elements/Content.js

```diff
@@ -84,12 +85,26 @@ const Content = ({
     }
   }, [containerHeight, textHeight]);
 
+  const randomIndex = React.useMemo(() => {
+    return Math.floor(Math.random() * listFeedColor.length);
+  }, []);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

While generating a random index for `listFeedColor` array, there's no check to ensure that the array is not empty. This could potentially lead to a division by zero error if `listFeedColor.length` is 0.

```diff
- return Math.floor(Math.random() * listFeedColor.length);
+ return listFeedColor.length > 0 ? Math.floor(Math.random() * listFeedColor.length) : 0;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->